package-lock.json

```diff
@@ -2275,6 +2275,21 @@
                 "graceful-fs": "^4.1.6"
             }
         },
+        "node_modules/@changesets/apply-release-plan/node_modules/prettier": {
+            "version": "2.8.8",
+            "resolved": "https://artifactory.g.devqa.gcp.dev.paypalinc.com/artifactory/api/npm/npm-all/prettier/-/prettier-2.8.8.tgz",
```

Comment:
I have no idea how this would have happened. there is an `.npmrc` file at the root of the project which should force you to use the public npm registry. somehow it used the private artifactory registry though.

do you have a public npm account? If so i would do the following
* `npm uninstall --save prettier`
* `npm login` <- use your public account creds
* `npm install --save prettier@3`